trade-finance-manager-api/src/v1/controllers/feedback-controller.js

```diff
@@ -38,7 +38,11 @@ exports.create = async (req, res) => {
     howCanWeImprove,
     emailAddress,
     submittedBy,
-    created: getUnixTime(new Date())
+    created: getUnixTime(new Date()),
+    auditDetails: {
```

Comment:
There might not be any user logged in when submitting feedback, I have a few ideas on what to do here:
- set lastUpdatedBy: null
- set `lastUpdatedBySystem: true`
 - Have an exemption on this collection to exclude it from the change stream

Reply:
Do we have access to the user if they are logged in and leave feedback? Wonder if we should do our best to populate if they are logged in and it's possible, and then resort to the `noUserLoggedIn` solution if not.
I think we'll need to include this in the changeStream just for audit purposes so can't exclude it :(